daq/gcp.py

```diff
@@ -248,9 +248,33 @@ def register_offenders(self):
             else:
                 LOGGER.info('Ignoring user %s', user_email)
 
+    def get_reports_from_date_range(self, device: str, start=None, end=None):
+        """Combine test results from reports within a date range"""
+        if not self._firestore:
+            LOGGER.error('Firestore not initialized.')
+            return
+        LOGGER.info('Looking for reports...')
+        origins = self._firestore.collection(u'origin').stream()
+        for origin in origins:
+            query = origin.reference.collection('runid')\
```

Comment:
no need for line continuation

Reply:
This line continuation is actually needed so I’m gonna leave it here